gap/FpCategories.gi

```diff
@@ -932,7 +925,7 @@ InstallMethod( MorphismInFpCategory,
 end );
 
 ##
-InstallMethod( \/,
+InstallOtherMethod( \/,
         [ IsPath, IsFpCategory ],
```

Comment:
The tests are failing because `IsQuiverVertex` implies `IsPath`, so after removing the method for `IsQuiverVertex` above, the method here is triggered instead of the generic one in CAP. Is there a more special filter which gives `IsPath` without `IsQuiverVertex`?

Reply:
I don't know :(

Reply:
In Algebroids.gi there are two filters used for this: `IsPathAlgebraElement` and `IsQuotientOfPathAlgebraElement`. Would those give what you want here?

Reply:
Probably yes. These are the filters for the two sorts of path algebras in QPA(2).

Reply:
I just checked, it does not work. So reverting to the old solution.